lua/cmake/init.lua

```diff
@@ -69,7 +69,7 @@ function cmake.run(...)
   end
 
   vim.list_extend(args, { ... })
-  return utils.run(target.filename, args, { cwd = target_dir.filename })
+  return utils.run(target.filename, args, { cwd = target_dir.filename, quickfix_only_on_error = false })
```

Comment:
Oh, makes sense. But could we rename `quickfix_only_on_error` param here into just `open_quickfix`?

Reply:
Yeah, that makes more sense, done